[Fundamentals] 'Cancel Cloudflare subscriptions' KB migration and typo fix (#4922)
cloudflare/cloudflare-docs: content/fundamentals/get-started/basic-tasks/account-maintenance/cancel-subscription.md

---
pcx-content-type: how-to

> title: Cancel Cloudflare Subscriptions

Reviewer: ```suggestion
title: Cancel Cloudflare subscriptions
```

weight: 2
---

# Cancel Cloudflare subscriptions

Cancellations are not processed until the end of the billing period, so you can continue using the add-on or subscription until the new billing period begins. To avoid unwanted charges, start the cancellation process before the end of your billing period.

---

## Step 1: Disable the Cloudflare subscription

To disable a subscription:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).

2. In the left sidebar, select **Websites**.

> 3. Select the domain for which you would like to cancel your subscription.

Reviewer: Any reason why who chose this workflow instead of what we normally have on the dev docs (Log in to the Cloudflare dashboard and nd select your account and website.)?

> 4. Select the feature name under **Active Subscriptions** (Free or Pro customers) or **Plan Extensions** (Enterprise customers).

Reviewer: ```suggestion
4. Select the feature you want to disable under **Active Subscriptions** (Free or Pro customers) or **Plan Extensions** (Enterprise customers).
```

> 5. Follow the instructions to disable the feature. Each feature has a different process which could include toggling a switch, clicking a button, or selecting 'Disable' from a drop-down list.

Reviewer: ```suggestion
5. Follow the instructions to disable the feature. Each feature has a different process which could include toggling a switch, clicking a button, or choosing _Disable_ from a drop-down list.
```

Reviewer: Drop-down options are italicized

## Step 2: Cancel the subscription in your billing profile

> 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).

Reviewer: ```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
```

Reviewer: Without this I added, your instructions workflow don't work.

> 2. In the left sidebar, select **Manage account** > **Billing**.

Reviewer: ```suggestion
2. Select **Manage account** > **Billing**.
```

Reviewer: We should avoid telling where in the screen customers should look, as this can become obsolete quite quickly

3. Select **Subscriptions**.

> 4. Select **Cancel** on the subscription you would like to disable.

Reviewer: ```suggestion
4. Find the subscription you want to disable and select **Cancel**.
```

Reviewer: It's better to start with the action people need to perform, and then telling them what they can do. Also, whenever possible use the present.

5. You will be asked to share feedback with us. Choose all that apply. 

6. Select **Confirm** to finish.

{{<Aside type="note">}}

> To alter your plan, select **Edit** and choose a new plan type. If you would like to cancel your paid plan, select **Free**.

Reviewer: I'm not sure where I can perform this action, and customers won't to as well. Is this on the same workflow where you click Cancel?
